### PR TITLE
pull images with cve fixes.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -411,9 +411,8 @@ done
 
 # this is the patched images which AKS are using.
 AKS_IP_MASQ_AGENT_VERSIONS="
-2.5.0
-2.5.0.1
 2.5.0.2
+2.5.0-linux-amd64.3
```
Comment (AbelHu) [inline vhdbuilder/packer/install-dependencies.sh:415]: Do you plan to add this in MCR repo? I only can find the latest version is v2.5.0.2 in MCR repo?

Comment (andyliuliming) [reply vhdbuilder/packer/install-dependencies.sh]: yes, 2.5.0.3 & 2.5.0-linux-amd64.3 is pushed in mcr by Chauncy.

docker pull mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.3
v2.5.0.3: Pulling from oss/kubernetes/ip-masq-agent
Digest: sha256:c9871244905cd5bd4c932b2abc911c77cdc8f9e5f18e73858ec1e3cdd17e38b6
Status: Downloaded newer image for mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.3
mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.3


docker pull mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0-linux-amd64.3
v2.5.0-linux-amd64.3: Pulling from oss/kubernetes/ip-masq-agent
Digest: sha256:e936bc72479e1ed06356d644dd4415481a18920f18a9eafc6b21ed878a16c231
Status: Image is up to date for mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0-linux-amd64.3
mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0-linux-amd64.3

**Comment by adilliadil** [comment]
Please add PR description

**Comment by paulgmiller** [comment]
Looks good but I don't have merge permissions.

**Review by adilliadil — COMMENTED** [review]
Please add PR description, otherwise LGTM

**Review by AbelHu — APPROVED** [review]

**Pull request merged**